src/main/kotlin/no/nav/familie/ef/sak/fagsak/FagsakService.kt

```diff
@@ -86,6 +86,9 @@ class FagsakService(
     fun finnFagsak(personIdenter: Set<String>, stønadstype: StønadType): Fagsak? =
         fagsakRepository.findBySøkerIdent(personIdenter, stønadstype)?.tilFagsakMedPerson()
 
+    fun finnFagsak(personIdenter: Set<String>): List<Fagsak?> =
```

Comment:
Denne returnerer alltid en not null? 
```suggestion
    fun finnFagsaker(personIdenter: Set<String>): List<Fagsak> =
```